ambari-web/app/views/common/configs/widgets/slider_config_widget_view.js

```diff
@@ -693,6 +692,7 @@ App.SliderConfigWidgetView = App.ConfigWidgetView.extend({
       unitLabel = units[unitLabelIndex];
       valueLabel = this._extraRound(tick);
     }
+    var separator = (separator && unitLabel) ? separator : '';
     label = valueLabel + separator + unitLabel;
```

Comment:
It is not a good practice to create a shadow variable in the context. Instead you can create a new variable (eg.: `localSeparator = separator;` or even better to skip the value assignments by doing the following:
```
label = valueLabel + ((separator && unitLabel) ? separator : '') + unitLabel;
```